Solution to resolve SC startup issue (#4837)
Particular/docs.particular.net: servicecontrol/troubleshooting.md

 1. If having issues remotely connecting to ServiceControl. Verify that firewall settings do not block access to the ServiceControl port specified in the URL.

NOTE: Before changing firewall setting to expose ServiceControl read [Securing ServiceControl](securing-servicecontrol.md).

### Method not found: 'Void System.Net.Http.Formatting.BaseJsonMediaTypeFormatter.set_SerializerSettings(Newtonsoft.Json.JsonSerializerSettings)'

If the following exception occurs at startup than this is likely because there are one or more versions of `Newtonsoft.Json` registered in the Global Assembly Cache (GAC).

```txt
Service cannot be started. System.MissingMethodException: Method not found: 'Void System.Net.Http.Formatting.BaseJsonMediaTypeFormatter.set_SerializerSettings(Newtonsoft.Json.JsonSerializerSettings)'.
```

> Removing these entries from the GAC should resolve this problem. 

Reviewer: ```suggestion
```

> 

Reviewer: ```suggestion
```

> These can be removed by running the following in an elevated (administrator) console:

Reviewer: ```suggestion
This problem can be resolved by removing `Newtonsoft.Json` entries from the GAC. It can be done with the following command in an elevated (administrator) console:
```

```cmd
gacutil /u Newtonsoft.Json
```

> It might be required to remove all `HKEY_LOCAL_MACHINE\SOFTWARE\Classes\Installer\Assemblies\Global Newtonsoft.Json` keys from the registry to be able to remove it with `gacutil /u Newtonsoft.Json`.

Reviewer: ```suggestion
It may be required to first remove all `HKEY_LOCAL_MACHINE\SOFTWARE\Classes\Installer\Assemblies\Global Newtonsoft.Json` keys from the registry before using `gacutil /u Newtonsoft.Json`.
```
